cdc/owner/ddl_sink.go

```diff
@@ -54,7 +54,6 @@ type DDLSink interface {
 	// the DDL event will be sent to another goroutine and execute to downstream
 	// the caller of this function can call again and again until a true returned
 	emitDDLEvent(ctx context.Context, ddl *model.DDLEvent) (bool, error)
-	emitBootstrapEvent(ctx context.Context, ddl *model.DDLEvent) error
```

Comment:
Why remove this method?

Reply:
It's removed, since the bootstrap event is going to be sent by the processor.